scripts/protocgen.sh

```diff
@@ -5,6 +5,6 @@ set -eo pipefail
 proto_dirs=$(find . -path ./third_party -prune -o -name '*.proto' -print0 | xargs -0 -n1 dirname | sort | uniq)
 for dir in $proto_dirs; do
   protoc -I. \
-  --gocosmos_out=Mgoogle/protobuf/timestamp.proto=github.com/golang/protobuf/ptypes/timestamp,Mgoogle/protobuf/duration.proto=github.com/golang/protobuf/ptypes/duration,plugins=grpc,paths=source_relative:. \
+  --gocosmos_out=Mgoogle/protobuf/timestamp.proto=github.com/golang/protobuf/ptypes/timestamp,Mgoogle/protobuf/duration.proto=github.com/golang/protobuf/ptypes/duration,plugins=grpc+interfacetype,paths=source_relative:. \
```

Comment:
will change this to `gogoslick` in a different pr as it will change all generated files